astronomer/providers/amazon/aws/example_dags/example_batch.py

```diff
@@ -211,7 +211,9 @@ def update_job_queue_func() -> None:
     client = boto3.client("batch")
     try:
         client.update_job_queue(jobQueue=JOB_QUEUE, state="DISABLED")
-
+        while get_job_queue_status() != "VALID":
```

Comment:
Why is the condition check for VALID and not DISABLED ? What are the job queue states? Looking at the [doc](https://docs.aws.amazon.com/batch/latest/userguide/job_queue_parameters.html), looks like they are ENABLED/DISABLED.

Also, since this function is to disable the job queue, does it make sense to rename the method to `disable_job_queue_func` from `update_job_queue_func` (and update the docstring) as it will more clearly indicate the purpose of the method?

Reply:
The state is changed to `DISABLED` but the status of it will be in `UPDATING` and then to `VALID`.  So I am getting the status of the job rather than the state for making sure the `DISABLED` state is updated with the status `VALID`.
<img width="868" alt="image" src="https://user-images.githubusercontent.com/92459020/177295382-3997ede3-cb44-41ef-9c62-672d33784223.png">

Reply:
The method used by AWS for disable or enable for job_queue was `client.update_job_queue(jobQueue=JOB_QUEUE, state="DISABLED").` so reciprocated the same to our method as well.

Reply:
ok sure will update it.